### PR TITLE
Add common plugin configuration

### PR DESCRIPTION
**Pull request by matt-rich**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -14,7 +14,79 @@
...
 
+    <!--
+    ============
+    ORGANISATION
+    ============
+    -->
+    <organization>
+        <name>Ministry of Justice</name>
+        <url>http://www.justice.gov.uk/</url>
+    </organization>
```
Comment (purple52) [inline pom.xml]: Organization should be right at the top in the super POM, I think.

File: pom.xml [hunk 2]
```diff
@@ -14,7 +14,79 @@
...
+        <plugins.maven.jar.version>2.6</plugins.maven.jar.version>
+        <plugins.maven.resources.version>2.7</plugins.maven.resources.version>
+        <plugins.maven.site.version>3.4</plugins.maven.site.version>
+        <plugins.maven.javadoc.version>2.10.4</plugins.maven.javadoc.version>
+        <plugins.maven.source.version>3.0.1</plugins.maven.source.version>
+
+        <plugins.maven.wagon.version>2.10</plugins.maven.wagon.version>
+        <plugins.maven.war.version>2.6</plugins.maven.war.version>
+        <plugins.versions.version>2.2</plugins.versions.version>
+        <liquibase.version>3.4.2</liquibase.version>
```
Comment (purple52) [inline pom.xml]: This is duplicated further down

File: pom.xml [hunk 3]
```diff
@@ -46,4 +148,580 @@
...
+                                <resources>
+                                    <resource>
+                                        <directory>src/raml/json/schema</directory>
+                                        <filtering>true</filtering>
+                                    </resource>
+                                </resources>
+                            </configuration>
+                        </execution>
+                    </executions>
+                </plugin>
```
Comment (purple52) [inline pom.xml]: The maven-resources-plugin config is project-specific - it should be left in the framework pom

File: pom.xml (hunk 1 above)
Comment (matt-rich) [inline pom.xml]: Agreed, will move up.

File: pom.xml (hunk 2 above)
Comment (matt-rich) [inline pom.xml]: Removed.

File: pom.xml (hunk 3 above)
Comment (matt-rich) [inline pom.xml]: Removed and reverted (non-pushed) change in other repository.

**Pull request merged**